scrapers/AbuseMe.yml

```diff
@@ -0,0 +1,41 @@
+name: AbuseMe
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - abuseme.com/video
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class="shoot-title"]/text()
+      Date:
+        selector: //div[@class="playerHdr_rgt"]/text()
+        postProcess:
+          - replace:
+            - regex: (\w+:\s)(.+)
+              with: $2
+          - parseDate: Jan 2, 2006
+      Details: //div[@class="playerTxt"]
+      Tags:
+        Name: 
+            selector: //meta[@http-equiv="keywords"]/@content
+            split: ","
+            # Bad way to get tags
+      Performers:
```

Comment:
the performers seem to fail if more than one are present in the scene eg `http://abuseme.com/video3405615/spicy-j-likes-it-rough`
using
```yaml
        Name:
          selector: //script[contains(.,"bbGaParams.shootModels")]/text()
          postProcess:
            - replace:
                - regex: .+\[(".+").\].+
                  with: $1
                - regex: ("\d+\s:\s)([^"]+)(")
                  with: $2
          split: ","
```
seems ok